ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -249,6 +258,103 @@ export default {
         },
       },
     },
+    {
+      id: 'DelubrumSav Guard Fiery Portent',
+      netRegex: NetRegexes.startsUsing({ source: 'Queen\'s Soldier', id: '583F', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Soldat Der Königin', id: '583F', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Soldat De La Reine', id: '583F', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'クイーンズ・ソルジャー', id: '583F', capture: false }),
+      delaySeconds: 4,
+      response: Responses.stopEverything('alert'),
```

Comment:
```suggestion
      response: Responses.stopEverything('alarm'),
```

What about making this one alarm and keeping icy as an alert, so that the sounds are differentiated?

Reply:
Pyretic ticks hit for 100k+, even with some mitigations. One tick is survivable if you are at full hp. However, I will make this adjustment as in my experience people tend to die from the Pyretic tick(s) if they get hit by it.